src/beacon-api/app/websockets/manager.py

```
@@ -0,0 +1,108 @@
+        """Listens to session:channel:{session_id} in Redis and forwards to local sockets."""
+        channel_name = f"session:channel:{session_id}"
+        pubsub = redis_service.client.pubsub()
+        await pubsub.subscribe(channel_name)
+        try:
+            async for message in pubsub.listen():
+                if message["type"] == "message":
+                    data = json.loads(message["data"])
+                    await self.send_to_session_local(session_id, data)
+        except asyncio.CancelledError:
+            logger.info(f"Subscription task for session {session_id} cancelled.")
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)